src/util/tr.js

```diff
@@ -499,6 +496,29 @@ const _tr = (msg, {
       const body = "<div class='wzrkPPdscr' style='color:" + textColor + "'>" + descriptionText + '<div></td></tr></table></div>'
       html = css + title + body
     }
+    if (displayObj.preview && displayObj['custom-editor']) {
+      iframe.sandbox = 'allow-scripts allow-popups allow-popups-to-escape-sandbox'
+      /* eslint-disable no-cond-assign */
+      var properties = {}
```

Comment:
use `let` please

Reply:
Sure

Reply:
declare this variable outside the if scope to make it available on line 538

Reply:
Done